2024-06-03.mdx

```diff
@@ -0,0 +1,55 @@
+---
+date: 6-3-2024
+---
+
+### What's new with Xata
+
+**UI improvements**
+
+You might have noticed already, but we've been beautifying the UI in the past weeks. For example, check out this empty state page for a new database: 🤩
+
+![Empty state for the new database page](/images/2024-06-03-empty-state.png)
+
+Other UI improvements:
+
+* The Playground now supports running multiple statements at once.
+* The branch selector got an upgrade and now works better with looooong branch names.
+* Fixed using `now()` as a default value for date columns.
+
+
+**Postgres compatibility improvements**
+
+Also the past couple of weeks we've seen a ton of improvements to our compatibility with the Postgres wire protocol:
+  * Improved compatibility with more tools that use the Postgres wire protocol, including JetBrains DataGrip.
+  * Added support for several system info functions.
+  * Allow locking clauses in SELECT statements. For example, the following is now supported: `SELECT * FROM table1 WHERE name = 'John' FOR UPDATE`.
+  * Improved compatibility and fixed issues when running [Ghostfolio](https://github.com/ghostfolio/ghostfolio) with Xata.
+  * Reduced clutter and errors during pg_restore runs.
+  * Added support for `SHOW session_replication_role`.
+
+**Extensions!**
+
+We added support for the first extensions on shared clusters:
```

Comment:
Might be worth clarifying that this is only for the public beta (and a link to the docs?)